app/src/main/java/com/eveningoutpost/dexdrip/Models/Calibration.java

```diff
@@ -896,6 +898,10 @@ public static void adjustRecentBgReadings(int adjustCount) {
             try {
                 final Calibration latestCalibration = Calibration.lastValid();
                 int i = 0;
+                boolean uploadModified = Pref.getBoolean("upload_modified_bgreadings", false);
```

Comment:
We currently use `adjusted` in our naming throughout this file. I'm concerned that introducing a new word `modified` may cause confusion to other maintainers.

What do you think about using `adjusted` instead of `modified` for these new additions? That way we can align with existing naming conventions

Reply:
Happy with alignment, will make changes :)

Reply:
Changes made, still need to test - will be building shortly and will start running immediately.